### DISCUSSION TITLE
Is it possible to store snapshot data in to a different folder?

### DISCUSSION THREAD

**Discussion by johnnydev543** [erigontech/erigon#4331] [Ideas]
If erigon has this feature, that will be great that the snapshot folder can be store in HDD to save the SSD space.

**Comment by AskAlexSharov** [comment]
done. snapshots are in: `<your_datadir>/snapshots`
use our alpha releases and flag --syncmode=snap

**Reply by AskAlexSharov** [reply]
same about `<your_datadir>/eth-temp` folder

**Reply by AskAlexSharov** [reply]
main db is in `<your_datadir>/chaindata`, but easier mount `<your_datadir>` to fast drive, and `<your_datadir>/snapshots` to slow drive.

**Comment by johnnydev543** [comment]
Thx for the reply.
Unfortunately, I'm using docker.so symlink doesn't work for this.
`panic: rename /datadir/snapshots_tmp /datadir/snapshots: not a directory`
Is there a flag to support custom folder path?

**Reply by AskAlexSharov** [reply]
ah, i forgot about this place... we must change /snapshots_tmp to /snapshots/tmp
so, in your case: let Erigon download all snapshot  - it will rename folder in the end. 
Then stop erigon, move move  /datadir/snapshots dir to another drive, mount that dir to /datadir/snapshots. Erigon will not do any renaming in future. 

I will try to fix it tomorrow.

**Reply by AskAlexSharov** [reply]
done by https://github.com/ledgerwatch/erigon/pull/4349
